it/v5/createForeignPropertyPeriodCumulativeSummary/def1/Def1_CreateForeignPropertyPeriodCumulativeSummaryISpec.scala

```diff
@@ -0,0 +1,267 @@
+        response.body shouldBe ""
+        response.header("Content-Type") shouldBe None
+      }
+    }
+
+    "return bad request error" when {
+      "badly formed json body" in new TysIfsTest {
+        override def setupStubs(): StubMapping = {
+          AuditStub.audit()
+          AuthStub.authorised()
+          MtdIdLookupStub.ninoFound(nino)
+        }
+        val response: WSResponse = await(request().addHttpHeaders(("Content-Type", "application/json")).put("{ badJson }"))
+        response.json shouldBe Json.toJson(BadRequestError)
+        response.status shouldBe BAD_REQUEST
+      }
+    }
+
+    "return error according to spec" when {
+      "validation error" when {
+        def validationErrorTest(requestNino: String,
+                                requestBusinessId: String,
+                                requestTaxYear: String,
+                                requestBody: JsValue,
+                                expectedStatus: Int,
+                                expectedBody: MtdError): Unit = {
+          s"validation fails with ${expectedBody.code} error" in new TysIfsTest {
+
+            override val nino: String       = requestNino
+            override val businessId: String = requestBusinessId
+            override val mtdTaxYear: String = requestTaxYear
+
+            override def setupStubs(): StubMapping = {
+              AuditStub.audit()
+              AuthStub.authorised()
+              MtdIdLookupStub.ninoFound(nino)
+            }
+
+            val response: WSResponse = await(request().put(requestBody))
+            response.json shouldBe Json.toJson(expectedBody)
+            response.status shouldBe expectedStatus
+          }
+        }
+        val input = List(
+          ("AA1123A", "XAIS12345678910", "2025-26", requestBody, BAD_REQUEST, NinoFormatError),
+          ("AA123456A", "XA***IS1", "2025-26", requestBody, BAD_REQUEST, BusinessIdFormatError),
+          ("AA123456A", "XAIS12345678910", "20256", requestBody, BAD_REQUEST, TaxYearFormatError),
+          ("AA123456A", "XAIS12345678910", "2025-27", requestBody, BAD_REQUEST, RuleTaxYearRangeInvalidError),
+          ("AA123456A", "XAIS12345678910", "2024-25", requestBody, BAD_REQUEST, RuleTaxYearNotSupportedError),
+          (
+            "AA123456A",
+            "XAIS12345678910",
+            "2025-26",
+            requestBodyWith(entryWithCustomNumber("premisesRunningCosts", Some(1.234))),
+            BAD_REQUEST,
+            ValueFormatError.copy(paths = Some(List("/foreignProperty/0/expenses/premisesRunningCosts")))),
+          (
+            "AA123456A",
+            "XAIS12345678910",
+            "2025-26",
+            requestBodyWith(invalidEntryWithConsolidatedExpenses()),
+            BAD_REQUEST,
+            RuleBothExpensesSuppliedError.copy(paths = Some(List("/foreignProperty/0/expenses")))),
+          ("AA123456A", "XAIS12345678910", "2025-26", JsObject.empty, BAD_REQUEST, RuleIncorrectOrEmptyBodyError),
```

Comment:
Business spec is incorrect/out of date for incorrect body, see BVR E8. See if you need a test to check for emptiness where we actually submit an empty body.